eng/tools/openapi-diff-runner/src/detect-breaking-change.ts

```diff
@@ -355,35 +362,49 @@ export function isInDevFolder(swaggerPath: string) {
 }
 
 /**
+ * Find the path to the closest readme.md file based on the input file path,
+ * searching upward from the file's directory up to the 'resource-manager' or 'data-plane' sub path.
  * Example:
  * input: specification/network/resource-manager/Microsoft.Network/stable/2019-11-01/network.json
- * returns: specification/network/resource-manager
+ * returns: path to the directory containing the closest readme.md file, up to specification/network/resource-manager
  */
 export function getReadmeFolder(swaggerFile: string) {
   const segments = swaggerFile.split(/\\|\//);
 
-  if (segments && segments.length >= 3) {
-    // Handle dev folder conversion
-    if (segments[0] === "dev") {
-      segments[0] = "specification";
-    }
+  if (!segments || segments.length < 3) {
+    return undefined;
+  }
 
-    // Look for "resource-manager" or "data-plane" in the path
-    const resourceManagerIndex = segments.findIndex((segment) => segment === "resource-manager");
-    if (resourceManagerIndex !== -1) {
-      return segments.slice(0, resourceManagerIndex + 1).join("/");
-    }
+  // Handle dev folder conversion
```

Comment:
Fine for now, but please open an issue to remove this.  We shouldn't have any special code for the "dev" folder.

Reply:
https://github.com/Azure/azure-sdk-tools/issues/11335